scalafix-rules/src/main/scala-2/scalafix/internal/pc/ScalafixGlobal.scala

```diff
@@ -72,6 +73,13 @@ class ScalafixGlobal(
     import scala.meta.internal.semanticdb.Scala._
     if (!symbol.isGlobal) return Nil
 
+    def EncodedTermName(str: String): TermName = TermName(
+      NameTransformer.encode(str)
+    )
+    def EncodedTypeName(str: String): TypeName = TypeName(
+      NameTransformer.encode(str)
```

Comment:
I wonder if inverse lookup of types is actually needed for ExplicitResultTypes (it might be a lot of code that was copied from metals)